app/instance/subsystem.go

```diff
@@ -23,36 +23,50 @@ import (
 	"github.com/percona/pmm/proto"
 )
 
+const (
+	SubsystemOS = iota + 1
+	SubsystemAgent
+	SubsystemMySQL
+	SubsystemMongo
+)
+
+const (
+	SubsystemNameOS    = "os"
+	SubsystemNameAgent = "agent"
+	SubsystemNameMySQL = "mysql"
+	SubsystemNameMongo = "mongo"
+)
+
 var subsysName map[uint]string = map[uint]string{
-	1: "os",
-	2: "agent",
-	3: "mysql",
-	4: "mongo",
+	SubsystemOS:    SubsystemNameOS,
+	SubsystemAgent: SubsystemNameAgent,
+	SubsystemMySQL: SubsystemNameMySQL,
+	SubsystemMongo: SubsystemNameMongo,
 }
```

Comment:
I don't like those hard coded values for subsystems at all, but now at least they are constants so harder to make mistakes with type checking and tests.